sysutils/pfSense-pkg-syslog-ng/files/usr/local/www/syslog-ng_log_viewer.php

```diff
@@ -54,118 +54,111 @@
+					<?php
+					$log_files = syslogng_get_log_files($objects);
+					foreach ($log_files as $log_file) {
+						if ($log_file == $logfile) {
+							echo "<option value=\"$log_file\" selected=\"selected\">$log_file</option>\n";
+						} else {
+							echo "<option value=\"$log_file\">$log_file</option>\n";
+						}
 					}
-				}
-				?>
+					?>
 				</select></td></tr>
-				<tr><td class="border-bottom" width="22%">Limit</td><td class="border-bottom" width="78%"><select name="limit">
-				<?php
-				$limit_options = array("10", "20", "50", "100", "250", "500");
-				foreach($limit_options as $limit_option) {
-					if($limit_option == $limit) {
-						echo "<option value=\"$limit_option\" selected=\"selected\">$limit_option</option>\n";
-					} else {
-						echo "<option value=\"$limit_option\">$limit_option</option>\n";
+				<tr><th width="22%">Limit</th><td width="78%"><select name="limit">
+					<?php
+					$limit_options = array("10", "20", "50", "100", "250", "500");
+					foreach ($limit_options as $limit_option) {
+						if ($limit_option == $limit) {
+							echo "<option value=\"$limit_option\" selected=\"selected\">$limit_option</option>\n";
+						} else {
+							echo "<option value=\"$limit_option\">$limit_option</option>\n";
+						}
 					}
-				}
-				?>
+					?>
 				</select></td></tr>
-				<tr><td class="border-bottom" width="22%">Include Archives</td><td class="border-bottom" width="78%"><input type="checkbox" name="archives" <?php if($archives) echo " CHECKED"; ?> /></td></tr>
-				<tr><td class="border-bottom" width="22%">Filter</td><td class="border-bottom" width="78%"><input name="filter" value="<?=$filter?>" /></td></tr>
-				<tr><td class="border-bottom" width="22%">Inverse Filter (NOT)</td><td class="border-bottom" width="78%"><input type="checkbox" name="not" <?php if($not) echo " CHECKED"; ?> /></td></tr>
-				<tr><td class="border-bottom" colspan="2"><input type="submit" value="Refresh" /></td></tr>
-				<tr><td class="border-bottom" colspan="2">
-				<table class="tabcont" width="100%" border="0" cellspacing="0" cellpadding="0">
-				<?php
-				if(!empty($log_messages)) {
-					echo "<tr><td class=\"listtopic\">Showing $log_messages_count of $log_lines messages</td></tr>\n";
-					foreach($log_messages as $log_message) {
-						echo "<tr><td class=\"listr\">$log_message</td></tr>\n";
-					}
-				} else {
-					echo "<tr><td><span class=\"red\">No log messages found or log file is empty.</span></td></tr>\n";
-				}
-				?>
-				</table>
+				<tr><th width="22%">Include Archives</th><td width="78%"><input type="checkbox" name="archives" <?php if($archives) echo " CHECKED"; ?> /></td></tr>
+				<tr><th width="22%">Filter</th><td width="78%"><input name="filter" value="<?=$filter?>" /></td></tr>
+				<tr><th width="22%">Inverse Filter (NOT)</th><td width="78%"><input type="checkbox" name="not" <?php if($not) echo " CHECKED"; ?> /></td></tr>
```

Comment:
Well, letting that handled automatically produces awful useless spacing in the form.